packs/equipment/injection-reservoir.json

```diff
@@ -9,7 +9,7 @@
         },
         "containerId": null,
         "description": {
-            "value": "<p>This reservoir and spring-loaded needle can be attached to a weapon to let it inject deadly poisons. Additionally, the reservoir can be filled with an injury poison. Immediately after a successful attack with the adjusted weapon, you can inject the target with the loaded poison by activating the reservoir with an Interact action. Refilling the reservoir with a new poison requires 3 Interact actions and uses both hands.</p>\n<p>Adding an injection reservoir to a weapon throws off its balance, causing the multiple attack penalty with the weapon to be one greater than usual (usually –6 on a second attack and –11 on a third; or –5 and –10 with an agile weapon).</p>"
+            "value": "<p>This reservoir and spring-loaded needle can be attached to a weapon to let it inject deadly poisons. Additionally, the reservoir can be filled with an injury poison. Immediately after a successful attack with the adjusted weapon, you can inject the target with the loaded poison by activating the reservoir with an Interact action. Refilling the reservoir with a new poison requires 3 Interact actions and uses both hands.</p>\n<p>Adding an injection reservoir to a weapon throws off its balance, causing the multiple attack penalty with the weapon to be one greater than usual (usually –6 on a second attack and –11 on a third; or –5 and –10 with an agile weapon).</p>\n<p>@UUID[Compendium.pf2e.equipment-effects.Item.Effect: Injection Reservoir]</p>"
```

Comment:
Instead of an effect this should be something that is handled by the new attachment stuff when it gets expanded.